infrastructure/src/main/java/org/corfudb/infrastructure/LogReplicationServer.java

```diff
@@ -123,9 +123,10 @@ private void handleLogReplicationNegotiationRequest(CorfuMsg msg, ChannelHandler
 
     @ServerHandler(type = CorfuMsgType.LOG_REPLICATION_QUERY_LEADERSHIP)
     private void handleLogReplicationQueryLeadership(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r) {
-        log.info("Log Replication Query Leadership Request received by Server.");
+        log.debug("Log Replication Query Leadership Request received by Server.");
```

Comment:
dint we want to make this trace?

Reply:
yes, but eventually when the product is a bit more stable. We said to leave it while we're still in early testing stage. But definitely something to downgrade to trace afterwards.